Index: accel-ppp-master/CMakeLists.txt
===================================================================
--- accel-ppp-master.orig/CMakeLists.txt
+++ accel-ppp-master/CMakeLists.txt
@@ -2,10 +2,6 @@ cmake_minimum_required(VERSION 2.6)
 
-if (CMAKE_BINARY_DIR STREQUAL CMAKE_HOME_DIRECTORY)
-    message(FATAL_ERROR "Building in source directory is forbidden. Please make separated build directory.")
-endif ()
-
 project(accel-ppp C)
Index: accel-ppp-master/accel-pppd/CMakeLists.txt
===================================================================
--- accel-ppp-master.orig/accel-pppd/CMakeLists.txt
+++ accel-ppp-master/accel-pppd/CMakeLists.txt
@@ -125,10 +125,3 @@ INSTALL(TARGETS accel-pppd
 
 INSTALL(FILES accel-ppp.conf.5 DESTINATION share/man/man5)
 
-IF (NOT DEFINED CPACK_TYPE)
-	INSTALL(FILES accel-ppp.conf DESTINATION "${CMAKE_INSTALL_SYSCONFDIR}" RENAME accel-ppp.conf.dist)
-
-	INSTALL(DIRECTORY DESTINATION "${CMAKE_INSTALL_LOCALSTATEDIR}/log/accel-ppp")
-	INSTALL(DIRECTORY DESTINATION "${CMAKE_INSTALL_LOCALSTATEDIR}/lib/accel-ppp")
-ENDIF (NOT DEFINED CPACK_TYPE)
-
